src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_CHAR = 'SORT_BY_CHAR';
+const SORT_BY_LENGTH = 'SORT_BY_LENGTH';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const Goods = ({ goods }) => (
```

Comment:
1) Move it to another components - `GoodsList`
2) Change the name of the component to `Item List`. I think it will be better that way
```suggestion
const GoodsList = ({ goods }) => (
```